src/main/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedProcessor.java

```diff
@@ -58,7 +61,16 @@ public void processQuestionnaireLinked(ResponseManagementEvent questionnaireLink
     UacQidLink uacQidLink = uacQidLinkOpt.get();
     Case caze = caseOpt.get();
 
+    // If already receipted
+    if (!uacQidLink.isActive()) {
```

Comment:
this is getting a bit confusing that we have a `receiptReceived` on the case and an `active` on the UAC. I would say that we should probably be checking the case `isReceiptReceived` and only flipping it to be true if it's set false, not this mixed version.